unstructured/partition/common.py

```diff
@@ -158,40 +167,48 @@ def normalize_layout_element(
             coordinates=coordinates,
             coordinate_system=coordinate_system,
             metadata=class_prob_metadata,
+            detection_origin=origin,
         )
         if element_type == "Headline":
             _element_class.metadata.category_depth = 1
         elif element_type == "Subheadline":
             _element_class.metadata.category_depth = 2
         return _element_class
     elif element_type == "Checked":
-        return CheckBox(
+        checkbox = CheckBox(
```

Comment:
why can't we keep the original pattern of just return instead of using a variable to capture the return then return?

Reply:
You're right, I revert to use that pattern